manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "rpi_power",
+  "name": "Raspberry Pi Power Supply Checker",
+  "documentation": "https://yourdomain.com",

```

Comment:
Should this url point (for better) to this github, README.md or something?

Reply:
Yepp ill resolve that

Reply:
Thanks @swetoast. 
I also confirm that it resolved it.
Thank you @nickm324 for the fast implementation.